olmeiro-orozco/tarea-5/js/helpers.js

```diff
@@ -0,0 +1,51 @@
+import { Caja } from "./caja.js";
+
+const cambiarCajas = (grid, caja1, caja2) => {
+  const temporal = grid[caja1.fila][caja1.columna];
+
+  grid[caja1.fila][caja1.columna] = grid[caja2.fila][caja2.columna];
+  grid[caja2.fila][caja2.columna] = temporal;
+};
+
+const getGridAleatorio = () => {
+  let grid = [
+    [1, 2, 3, 4],
+    [5, 6, 7, 8],
+    [9, 10, 11, 12],
+    [13, 14, 15, 0],
+  ];
+
+  let cajaBlanca = new Caja(3, 3);
+
+  for (let i = 0; i < 1000; i++) {
```

Comment:
@OlmeiroOrozco , you should use modern approach like map, forEach or reduce, also remember to name properly the variable i